api/observation.go

```diff
@@ -294,14 +294,14 @@ func (api *DatasetAPI) getObservationList(ctx context.Context, versionDoc *model
 	}
 
 	queryObject := observation.Filter{
-		InstanceID:       versionDoc.ID,
+		InstanceID:       "d413012e-d960-4faf-b6b9-731c4362975f", //versionDoc.ID,
```

Comment:
is this ID supposed to be hardcoded? Maybe you did it for testing, and then forgot to add `versionDoc.ID` back?

Reply:
Thanks, I hardcoded this for testing purposes, need to revert good spot